docs/cost/ids/core.md

```diff
@@ -0,0 +1,267 @@
+# Core standard
+
+<style>
+.wy-nav-content {
+  max-width: 1200px;
+}
+</style>
+
+## Project level
+
+### Identification
+
+```{csv-table-no-translate}
+:header-rows: 1
+:included_cols: 0,1,2
+:file: ../../../build/current_lang/project-level-identification.csv
+```
+
+### Preparation
+
+```{csv-table-no-translate}
+:header-rows: 1
+:included_cols: 0,1,2
+:file: ../../../build/current_lang/project-level-preparation.csv
+```
+
+### Project completion
+
+```{csv-table-no-translate}
+:header-rows: 1
+:included_cols: 0,1,2
+:file: ../../../build/current_lang/project-level-completion.csv
+```
+
+### Reactive disclosures
+
+#### Identification and preparation
+
+```{csv-table-no-translate}
+:header-rows: 1
+:included_cols: 0,1,2
+:file: ../../../build/current_lang/reactive-project-level-identification-preparation.csv
+```
+
+#### Completion
+
+```{csv-table-no-translate}
+:header-rows: 1
+:included_cols: 0,1,2
+:file: ../../../build/current_lang/reactive-project-level-completion.csv
+```
+
+#### Implementation progress reports
```

Comment:
This was originally one heading level lower than Completion

Reply:
Ah, good spot. Fixed in https://github.com/open-contracting/infrastructure/pull/422/commits/c71446fb5f86849bc7575b9811164ed53e27a159